client/src/main/python/slipstream/wrappers/BaseWrapper.py

```diff
@@ -22,11 +24,35 @@
 from slipstream.SlipStreamHttpClient import SlipStreamHttpClient
 from slipstream.NodeDecorator import NodeDecorator
 from slipstream.exceptions import Exceptions
+from slipstream.NodeInstance import NodeInstance
+from slipstream.exceptions.Exceptions import TimeoutException, ExecutionException
+
+
+class InconsistentScaleStateError(Exceptions.ExecutionException):
+    pass
+
+
+class InconsistentScalingNodesError(Exceptions.ExecutionException):
```

Comment:
Here is the result of the search, which suggests that neither scale CLIs nor their base class do not reference the exceptions.
```
SlipStreamClient konstan$ egrep -lr "InconsistentScalingNodesError|InconsistentScaleStateError" ./*
./client/src/main/python/slipstream/exceptions/Exceptions.py
./client/src/main/python/slipstream/executors/MachineExecutor.py
./client/src/main/python/slipstream/wrappers/BaseWrapper.py
./client/src/test/python/TestCloudWrapper.py
SlipStreamClient konstan$ find . -name "ss-scale*"                                                 
./client/src/main/python/ss-scale-disk.py
./client/src/main/python/ss-scale-resize.py
SlipStreamClient konstan$ 
```